kitten-heart/src/main/java/net/pixaurora/kitten_heart/impl/config/ScrobblerCache.java

```diff
@@ -75,7 +75,7 @@ public ScrobblerCache deserialize(JsonElement json, Type _type, JsonDeserializat
                 throws JsonParseException {
             ArrayList<Scrobbler> scrobblers = new ArrayList<>();
 
-            for (JsonElement scrobblerData : json.getAsJsonArray().asList()) {
+            for (JsonElement scrobblerData : json.getAsJsonArray()) {
```

Comment:
Why do you change this line? :eyes:

Reply:
I had to downgrade GSON since the version bundled with 1.17.0 doesn't have this method yet.

Reply:
I see!!! Thank you :beaver: